File: launchdarkly/java/src/main/java/oseg/launchdarkly_examples/PostMigrationSafetyIssuesExample.java
package oseg.launchdarkly_examples;

import com.launchdarkly.client.ApiException;
import com.launchdarkly.client.Configuration;
import com.launchdarkly.client.api.*;
import com.launchdarkly.client.auth.*;
import com.launchdarkly.client.JSON;
import com.launchdarkly.client.model.*;

import java.io.File;
import java.math.BigDecimal;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class PostMigrationSafetyIssuesExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        ((ApiKeyAuth) config.getAuthentication("ApiKey")).setApiKey("YOUR_API_KEY");

        var flagSempatch = new FlagSempatch();
        flagSempatch.instructions(List.of ());

        try
        {
            var response = new FeatureFlagsApi(config).postMigrationSafetyIssues(
                "projectKey_string", // projectKey
                "flagKey_string", // flagKey
                "environmentKey_string", // environmentKey
                flagSempatch
            );

            System.out.println(response);
        } catch (ApiException e) {
            System.err.println("Exception when calling FeatureFlagsApi#postMigrationSafetyIssues");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/PostMigrationSafetyIssuesExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class PostMigrationSafetyIssuesExample
{
    fun postMigrationSafetyIssues()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        val flagSempatch = FlagSempatch(
            instructions = listOf (),
        )

        try
        {
            val response = FeatureFlagsApi().postMigrationSafetyIssues(
                projectKey = "projectKey_string",
                flagKey = "flagKey_string",
                environmentKey = "environmentKey_string",
                flagSempatch = flagSempatch,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling FeatureFlagsApi#postMigrationSafetyIssues")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling FeatureFlagsApi#postMigrationSafetyIssues")
            e.printStackTrace()
        }
    }
}
